src/database_context.ts

```diff
@@ -37,7 +37,6 @@ export class DatabaseContext {
     public agnostic: Knex;
 
     constructor() {
-        if ([EnvType.CI].includes(process.env.NODE_ENV as EnvType)) return;
```

Comment:
Do we no longer need this early return for CI?

Reply:
This was introduced before we seeded the kpop_videos database during the test setup, so trying to create a connection to that db would fail.